spec/adapters/datatables_spec.rb

```diff
@@ -4,18 +4,36 @@
   before { Daru::View.table_library = :datatables }
   before(:each) do
     @data_array = [[1, 15], [2, 30], [4, 40]]
+    @string_array = ["daru", "view"]
```

Comment:
We must use `let` here. Can you do changes accordingly? [Refer](https://www.ombulabs.com/blog/rails/rspec/ruby/let-vs-instance.html)